automatic/freecad/README.md

```diff
@@ -1,7 +1,64 @@
+- built-in scripting framework. FreeCAD features a built-in Python interpreter, with an API that covers almost any part of the application, the interface, the geometry and the  representation of this geometry in the 3D viewer. The interpreter can run complex scripts as well as single commands; entire modules can be programmed completely in Python.
+- built-in Python console. The Python interpreter includes a console with syntax highlighting, autocomplete and a class browser. Python commands can be issued directly in FreeCAD and immediately return results, permitting script writers to test functionality on the fly, explore the contents of FreeCAD's modules and easily learn about FreeCAD internals.
+- mirrors user interaction. Everything the user does in the FreeCAD interface executes Python code, which can be printed on the console and recorded in macros.
+- full macro recording and editing capabilities. The Python commands issued when the user manipulates the interface can be recorded, edited if needed, and saved to be reproduced later.
+- compound (ZIP based) document save format. FreeCAD documents are saved with a .FCStd extension. The document can contain many different types of information such as geometry, scripts or thumbnail icons. The .FCStd file is itself a zip container; a saved FreeCAD file has already been compressed.
+- fully customizable/scriptable Graphical User Interface. The Qt-based interface of FreeCAD is entirely accessible via the Python interpreter. Aside from simple functions FreeCAD itself provides to workbenches, the entire Qt framework is accessible. The user may perform any operation on the GUI such as creating, adding, docking, modifying or removing widgets and toolbars.
+- thumbnailer. (currently only Linux systems) FreeCAD document icons show the contents of the file in most file manager applications such as Gnome's Nautilus.
+- modular MSI installer. FreeCAD's installer allows flexible installations on Windows systems. Packages for Ubuntu systems are also maintained.
+
+## Package parameters
+** Used only for Portable or Pre Releases Currently **
+- `/UnzipLocation` - This is the location inside the Chocolatey lib folder specific to this package.
+- `/WorkingDirectory` - This will use the same location as the Unziplocation unless designated to be different.
+- `/TargetPath` - This is the location of the FreeCAD.exe file when unziped from the archive file.
+- `/IconLocation` - Icon location is same as FreeCAD.exe file unless designated to be different.
+- `/Arguments` - This will allow you to specify any of the FreeCAD command line arguements.
+- `/ShortcutFilePath` - This will place the shortcut if wanted on your Desktop.
+- `/Shortcut` - This is set to place the Shortcut only if listed in the Package Parameters ( ex: /Shortcut ). The default is to not place any Shortcut.

```

Comment:
Most other packages place shortcut by  default and use `/NoShortcut` or `/NoDesktopShortcut`

Reply:
This parameter is only for the Portable and Developer versions for now. I will change it to be on by default with the `/NoShortcut` as the parameter